### PR TITLE
Security advisories

### PR DESCRIPTION
**Pull request by sbesson**
Follow-up of #103 and #112, this PR reorganizes the security section as follows:

- `security/` is now the landing page for all security related content
- `security/advisories` is the landing page for the security advisories
- `security/advisories/YYYY-SVn-xxxx` is the landing page for a given advisory

See also:
- https://jenkins.io/security/advisories/
- https://www.mozilla.org/en-US/security/advisories/
- https://www.vmware.com/uk/security/advisories.html

### CODE REVIEW

**Comment by sbesson** [comment]
See https://snoopycrimecop.github.io/www.openmicroscopy.org/security/

File: security/index.html [hunk 1]
```diff
@@ -1,82 +1,14 @@
 ---
 layout: news
-title: Security Vulnerabilities
+title: Security
 filename: security
-main-blurb: Known vulnerabilities, workarounds, and resolutions
```
Comment (hflynn) [inline security/index.html]: We probably want a new blurb here
"Keeping your data secure" ?

Comment (sbesson) [reply security/index.html]: Probably. I can push `Keeping your data secure` and leave others to open follow-up PRs here and in the following section.

**Comment by sbesson** [comment]
@hflynn: pushed another commit which renames the layout as `security-advisory` and fixes the link to link back to the advisories list and updates the main security page description.

I propose not to modify the content of the `/security/advisories/` page here as my goal is to get the final URLs in place for live deployment. We can discuss and update the content of this page as follow-up PRs. In addition to removing the `Affected versions` column, another consideration for this page might be to add another link `back to Security`.

**Comment by hflynn** [comment]
Yeah, I think a 'Back to Security' link will be needed too. But happy for that change to be in a follow up.

URLs all implemented as suggested so good to merge from me

**Comment by chris-allan** [comment]
My only request would be to restore the advisory dates to the table on the `/security/advisories/` page. Probably as an additional column.

**Comment by sbesson** [comment]
@chris-allan: :+1: the first column of the advisories page has now been updated accordingly.

**Pull request merged**